lib/eex/test/eex_test.exs

```diff
@@ -428,8 +422,9 @@ defmodule EExTest do
     end
 
     test "sets external resource attribute" do
-      assert EExTest.Compiled.__info__(:attributes)[:external_resource] ==
-               [Path.join(__DIR__, "fixtures/eex_template_with_bindings.eex")]
+      assert EExTest.Compiled.__info__(:attributes)[:external_resource] == [
+               Path.join(__DIR__, "fixtures/eex_template_with_bindings.eex")
+             ]
```

Comment:
The break is happening at an unexpected place.

Reply:
Oh I love how it formatted this (though I'd opt for the trailing comma).  The `]` at the end of a non-empty line when the opening `[` is always confusing to read to me, this formatting is *much* better to me overall as it looks like an opened scope across the lines (the inside of the list) unlike the `==` that appears to be dangling and confusing.